articles/anomaly-detection/guides/view-anomaly-detection-events.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to use tenant traffic log data to view anomaly detection events. 
+topics:
+    - security
+    - anomaly-detection
+contentType: how-to
+useCase: tenant-logs
+---
+
+# View Anomaly Detection Events
+
+Your tenant logs contain useful data that you can use to build charts to look at the profile of the traffic going through your tenant. This is helpful when evaluating anomaly detection activity. For example, you can look for the following events to determine if you're under attack:
+- Abnormal bursts in traffic to the login flow that result in errors (such as wrong username or password errors).
+- Abnormal bursts in traffic coming from IP locales that are not expected.
+
+These events tend to happen without much change to the rate of successful sign-ins.
+
+You can use your tenant log data `event` field to view tenant traffic data. We recommend building a daily histogram of failure events of the following types:
+
+| Event Code | Event |
+| -- | -- |
+| `f` | Failed login |
+| `fcoa` | Failed cross-origin authentication |
+| `feccft` | Failed exchange |
+| `fepft` | Failed exchange |
+| `fsa` | Failed silent authentication |
+| `fu` | Failed login (invalid email/username) |
+| `ip` | Source IP (Use in conjunction with `fu` event traffic to determine where the failure traffic is coming from) |
```

Comment:
There is no `ip` Event Code. This is a field within the logs.

Reply:
Removed